Kiosk/SettingsHelper.cs

```diff
@@ -611,6 +628,8 @@ public string[] AvailableCustomVisionApiEndpoints
             {
                 return new string[]
                 {
+                    CustomEndpointName,
```

Comment:
Please reverted these two lines as well.

Reply:
Removed the discussed endpoints